.gitignore

```diff
@@ -5,8 +5,6 @@
 # @see https://www.drupal.org/project/drupal/issues/3257119
 /docroot/core
 /vendor/
-/docroot/sites/*/settings.*.php
-/docroot/sites/*/services*.yml
```

Comment:
You still want to exclude `settings.local.php` and `services.local.yml`.

Reply:
good catch, fixing!

Reply:
The current pattern (`/docroot/sites/*/settings.*.php`) should allow `/docroot/sites/*/settings.php` while ignoring `/docroot/sites/*/settings.local.php`. See my comment in #73.